src/components/organization/edit/EditOrganizationForm.tsx

```diff
@@ -78,7 +86,12 @@ const EditOrganizationForm = ({ organization, caUnit }: Props) => {
         name="name"
         label={t('name')}
       />
-      <Sites form={form} sites={sites} />
+      <DynamicComponent
+        environmentComponents={{ cut: SitesCut }}
+        defaultComponent={Sites}
```

Comment:
J'aurais fait 
```suggestion
        defaultComponent={<Sites sites={sites} form={form as unknown as UseFormReturn<SitesCommand>}}/>
```
Ça permet d'avoir des props différentes entre les versions de composants

Reply:
Ah mais oui !!! C'est bcp mieux je vais modifier le composant en conséquence merci !!!!